### PR TITLE
Updated geocoding parser

### PR DESCRIPTION
**Pull request by rastaarh**
Updated geocoding parser to fetch live data from Google's libPhoneNumber library metadata resources. 

@paween

### CODE REVIEW

File: libPhoneNumber-GeocodingParser/Podfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
```
Comment (paween) [inline libPhoneNumber-GeocodingParser/Podfile:2]: Remove these lines

File: libPhoneNumber-GeocodingParser/libPhoneNumber-GeocodingParser.xcodeproj/xcshareddata/xcschemes/libPhoneNumber-GeocodingParser.xcscheme [hunk 2]
```diff
@@ -53,7 +53,7 @@
       <CommandLineArguments>
          <CommandLineArgument
             argument = "/Users/rastaar/Documents/geocoding"
```
Comment (paween) [inline libPhoneNumber-GeocodingParser/libPhoneNumber-GeocodingParser.xcodeproj/xcshareddata/xcschemes/libPhoneNumber-GeocodingParser.xcscheme:55]: hmm?

File: libPhoneNumber-GeocodingParser/Podfile (hunk 1 above)
Comment (rastaarh) [reply libPhoneNumber-GeocodingParser/Podfile:2]: Done.

File: libPhoneNumber-GeocodingParser/libPhoneNumber-GeocodingParser.xcodeproj/xcshareddata/xcschemes/libPhoneNumber-GeocodingParser.xcscheme (hunk 2 above)
Comment (rastaarh) [reply libPhoneNumber-GeocodingParser/libPhoneNumber-GeocodingParser.xcodeproj/xcshareddata/xcschemes/libPhoneNumber-GeocodingParser.xcscheme:55]: Got rid of that from the scheme.

**Pull request merged**